docs/reference/modules/node.asciidoc

```diff
@@ -104,10 +110,14 @@ To create a standalone master-eligible node, set:
 node.master: true <1>
 node.data: false <2>
 node.ingest: false <3>
+node.ml: false <4>
+xpack.ml.enabled: true <5>
```

Comment:
For a node that does not have X-Pack installed, these settings will cause node startup to fail. There needs to be a caveat to these settings so that users that are not using X-Pack do not copy/paste these into their settings and then get confused when startup fails. This comment applies throughout.